Project-Bootcamp/views/pvp-view.js

```diff
@@ -0,0 +1,64 @@
+var React = require ('react');
+var _ = require ('lodash');
+var $ = require ('jquery');
+var Button = require ('../components/button');
+var Table = require ('../components/table');
+var BlizzardAPI = require ('../querys/query');
+
+var PvpView = React.createClass ({
+
+    getInitialState: function () {
+
+        return {
+            pvp: []
+        }
+    },
+
+    render: function () {
+
+        return (
+            <div>
+                <div>
+                    <h3><u>Top 10 pvp leaderboard</u></h3>
+                    <p>Fill the bracket, and then press the button to checkout the pvp leaderboards.</p>
+                    <Input label="Bracket" type="text" onChange{this.handleInputChange.bind (this, 'bracket')} />
```

Comment:
Careful here! ``onChange`` is again not being assigned. Think also on breaking props onto a new method when you have more than two assigned.